src/devices/Common/Iot/Device/Graphics/BdfFont.cs

```diff
@@ -257,7 +266,8 @@ private void ReadGlyphsData(StreamReader sr)
                 span = sr.ReadLine().AsSpan().Trim();
                 if (!span.StartsWith(s_endChar, StringComparison.Ordinal))
                 {
-                    throw new InvalidDataException("The font data is not well formed. expected ENDCHAR tag in the beginning of glyoh data.");
+                    throw new InvalidDataException(
+                        "The font data is not well formed. expected ENDCHAR tag in the beginning of glyoh data.");
```

Comment:
```suggestion
                        "The font data is not well formed. expected ENDCHAR tag in the beginning of glyph data.");
```

Since we are changing this line